src/entities/comment/ui/CommentItem.tsx

```diff
@@ -0,0 +1,66 @@
+import { IComment } from '../model/type';
+import { Edit2, ThumbsUp, Trash2 } from 'lucide-react';
+import { Button } from '../../../shared/ui/components';
+import { HighlightText } from '../../../shared/ui/HighlightText';
+
+interface CommentItemProps {
+  // 댓글 데이터
+  comment: IComment;
+  // 하이라이트 처리 검색어
+  searchQuery: string;
+
+  // 댓글 좋아요
+  onLikeComment: (comment: IComment) => void;
+  // 댓글 수정
+  onEditComment: (comment: IComment) => void;
+  // 댓글 삭제
+  onDeleteComment: (comment: IComment) => void;
+}
+
+const CommentItem = ({
+  comment,
+  searchQuery,
+  onLikeComment,
+  onEditComment,
+  onDeleteComment,
+}: CommentItemProps) => {
+  return (
+    <div
+      key={comment.id}
+      className="flex items-center justify-between text-sm border-b pb-1"
+    >
+      <div className="flex items-center space-x-2 overflow-hidden">
+        <span className="font-medium truncate">{comment.user.username}:</span>
+        <span className="truncate">
+          {HighlightText(comment.body, searchQuery)}
```

Comment:
`HighlightText` 의 역할이 컴포넌트인지 함수인지 의도가 불분명 한 것 같아요

Reply:
앞글자 대문자로 바꾸신김에 컴포넌트 어떠세요
```suggestion
          <HighlightText text={comment.body} highlight={searchQuery) />
```